test/test.jwt.ts

```diff
@@ -830,3 +830,17 @@ it('getRequestHeaders populates x-goog-user-project for JWT client', async () =>
   );
   assert.strictEqual(headers['x-goog-user-project'], 'fake-quota-project');
 });
+
+it('should return an ID token for fetchIdToken', async () => {
+  const keys = keypair(1024 /* bitsize of private key */);
+  const jwt = new JWT({
```

Comment:
I think we should add a test that ensures that the refresh logic works for both a compute client and a JWT client.

Reply:
done!